### ISSUE TITLE
Nothing joins anymore

### ISSUE THREAD

**Issue by swoopgrandchamp** [Koenkk/zigbee2mqtt#20671]
### What happened?

Few days ago I bought a Tuya temperature sensor and I was not able to make it join my network, my though was that the sensor was faulty because everything else was working as expected.
3 days ago I noticed my Aqara temperature sensors stopped working and I've tried to make them start working again with no success.
- Have tried to just press their button to send new values, didn't work
- Have tried to force remove them and repair, didn't work
- Have tried to force remove a Tuya door sensor (which was working fine), got a confirmation that it was removed from the network but haven't been able to re-join them after this

My main coordinator is a zzh CC2652 with the latest firmware and I have a spare slae.sh CC2652RB which I've tried replacing with but it didn't work as well.
Have also updated both to latest firmware, no success.

Basically I don't have any message at all when I put devices on pairing mode.
I'm running zigbee2mqtt on docker and permit join is enabled.

I've no idea what the problem is.

### What did you expect to happen?

Devices to join or at least show an error message

### Zigbee2MQTT version

1.35.0

### Adapter firmware version

20230507

### Adapter

zzh and slaesh

**Comment by swoopgrandchamp** [comment]
Right, so I've tried to remove from the USB extension cable and plug the coordinator directly to the usb port and I'm now able to join devices.
Have plugged it back to the extension cable and managed to pair some other devices as well.
Something funky going on, but glad I now can get back my devices to work again.

Closing as it was solved by itself.
